core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -60,22 +62,19 @@ class TransactionStateManager(brokerId: Int,
 
   this.logIdent = "[Transaction Log Manager " + brokerId + "]: "
 
-  type WriteTxnMarkers = WriteTxnMarkerArgs => Unit
+  type SendTxnMarkersCallback = (String, Int, TransactionMetadata, TransactionResult) => Unit
```

Comment:
Hmm, as discussed previously, it's preferable to use a case class instead of many unnamed parameters.